concat-with-maps.js

```diff
@@ -62,31 +73,51 @@ ConcatWithMaps.prototype.build = function() {
     });
   }
 
-  this.addFiles(this.inputPaths[0], beginSection);
+  this.addFiles(beginSection);
 
-  if (this.footer) {
-    beginSection();
-    concat.addSpace(this.footer);
-  }
   if (this.footerFiles) {
     this.footerFiles.forEach(function(ff) {
       beginSection();
       concat.addFile(ff);
     });
   }
+
+  if (this.footer) {
+    beginSection();
+    concat.addSpace(this.footer + '\n');
```

Comment:
I'm not sure why this newLine was needed, IHMO it should be removed. Will investigate later.